book/src/external-snapshotter.md

```diff
@@ -10,6 +10,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
+[external-snapshotter v6.0.0](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v6.0.0) | [release-6.0](https://github.com/kubernetes-csi/external-snapshotter/tree/release-6.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.5.0) | - | k8s.gcr.io/sig-storage/csi-snapshotter:v6.0.0 | v1.20 | - | v1.22
```

Comment:
Can you update this to 6.0.1?  Also what about snapshot-controller and snapshot validation webhook versions?